### DISCUSSION TITLE
Attachments aren't displayed in Report Portal

### DISCUSSION THREAD

**Discussion by hager-yousri** [reportportal/reportportal#1915] [Q&A]
I'm using robotframework
Screenshots are created for failed test cases
Also I use ScreenCapLibrary to record videos for tests

I can see the attachments [screenshots - videos] in html report and log **locally**
**but  I can't see the attachments [screenshots - videos] on report portal**

**First** when opening any failed test case the attachments [screenshots - videos] are displayed as shown in the attached images and when i try to open them I got 404
<img width="254" alt="Screenshot" src="https://user-images.githubusercontent.com/69680400/209026871-66a30ac7-d733-4620-b99b-8147446028bc.png">
<img width="721" alt="Video" src="https://user-images.githubusercontent.com/69680400/209026905-5ab195d3-2463-4232-8964-abfd700b98c8.png">
<img width="346" alt="error" src="https://user-images.githubusercontent.com/69680400/209026918-a29340d0-bca8-4726-8e99-5bda19e9ea68.png">

**Second** I pass the html report and log to the report portal so whenever opening html report or log from log view then try to open any attachments [screenshots - videos] they can't be opened as well
<img width="503" alt="blocked" src="https://user-images.githubusercontent.com/69680400/209027056-c6767183-ea71-4bb1-8871-55d775bfd2a6.png">

**So how can I display the attachments [screenshots - videos] in report portal while using robot framework**

**Comment by HardNorth** [comment]
This looks like a problem with your server. Can you check `service-api` logs for errors?

**Reply by hager-yousri** [reply]
@HardNorth 
nothing helped me in the logs 
but I found this document https://github.com/reportportal/client-Python#send-attachement-screenshots  **but it is not clear what exactly should I add into my robot framework project to show the screenshots/videos in the report portal** 

I see the main problem with the attachment url which is displayed in report portal
**example for attachment path in report portal:** https://qcauto-rp.thiqah.sa/ui/recordedVideo_8.webm 
how can I fix this path to redirect correctly to the attachment
